enferno/admin/views.py

```
@@ -6588,3 +6591,513 @@
+        except ValueError:
+            return jsonify({"errors": [{"message": "Invalid limit or offset"}]}), 400
+        stmt = (
+            select(DynamicField).where(*filters).order_by(sort_clause).offset(offset).limit(limit)
+        )
+        count_stmt = select(func.count()).select_from(DynamicField).where(*filters)
+        total = db.session.execute(count_stmt).scalar_one()
+        items = db.session.execute(stmt).scalars().all()
+        data = [item.to_dict() for item in items]
+
+        # Simple entity_type filtering
+        entity_type_filter = request.args.get("entity_type")
```

Ruff (0.12.2)

Local variable `entity_type_filter` is assigned to but never used

Remove assignment to unused variable `entity_type_filter`

(F841)